src/test/rgw/bucket_notification/api.py

```diff
@@ -230,6 +230,32 @@ def del_config(self, notification=None):
 
 test_path = os.path.normpath(os.path.dirname(os.path.realpath(__file__))) + '/../'
 
+
+class BucketLogging:
```

Comment:
we don't need this.
bucket logging tests are done elsewhere (as part of s3test)